rust_src/src/marker.rs

```diff
@@ -24,3 +32,13 @@ impl LispMarkerRef {
 fn markerp(object: LispObject) -> LispObject {
     LispObject::from_bool(object.is_marker())
 }
+
+/// Return the position of MARKER, or nil if it points nowhere.
+#[lisp_fn]
+fn marker_position(object: LispObject) -> LispObject {
+    let pos = object.as_marker_or_error().charpos();
+    match pos {
+        None => LispObject::constant_nil(),
+        _ => LispObject::from_natnum(pos.unwrap() as EmacsInt),
```

Comment:
Yeah I know. I was just wondering if this might be ok in this case. But you're right.